packages/@aws-cdk/aws-codedeploy/test/lambda/test.custom-deployment-config.ts

```diff
@@ -0,0 +1,255 @@
+import { expect, haveResource, ResourcePart } from '@aws-cdk/assert';
+import * as lambda from '@aws-cdk/aws-lambda';
+import * as cdk from '@aws-cdk/core';
+import { Test } from 'nodeunit';
+import * as codedeploy from '../../lib';
+
+function mockFunction(stack: cdk.Stack, id: string) {
+  return new lambda.Function(stack, id, {
+    code: lambda.Code.fromInline('mock'),
+    handler: 'index.handler',
+    runtime: lambda.Runtime.NODEJS_10_X,
+  });
+}
+function mockAlias(stack: cdk.Stack) {
+  return new lambda.Alias(stack, 'Alias', {
+    aliasName: 'my-alias',
+    version: new lambda.Version(stack, 'Version', {
+      lambda: mockFunction(stack, 'Function'),
+    }),
+  });
+}
+
+export = {
+  'custom resource created'(test: Test) {
+    const stack = new cdk.Stack();
+    const application = new codedeploy.LambdaApplication(stack, 'MyApp');
+    const alias = mockAlias(stack);
```

Comment:
These 3 lines are repeated for every test... Can you put these in a `setUp`? [Like this](https://github.com/aws/aws-cdk/blob/19dd454c79b1be670b7bd7a4a99fcbde154f658a/packages/%40aws-cdk/aws-secretsmanager/test/test.secret-rotation.ts#L7-L25).

Reply:
Done